KsApi/Service.swift

```diff
@@ -34,6 +34,9 @@ public struct Service: ServiceType {
     self.language = language
     self.currency = currency
     self.buildVersion = buildVersion
+
+    // Global override required for injecting custom User-Agent header in ajax requests
+    UserDefaults.standard.register(defaults: ["UserAgent" : "\(Service.userAgent)"])
```

Comment:
Do you use `"\(Service.userAgent)"` to turn `Service.userAgent` into a string?

Reply:
😄 indeed! thanks